src/command_modules/azure-cli-role/azure/cli/command_modules/role/custom.py

```diff
@@ -34,6 +37,8 @@
 
 _CUSTOM_RULE = 'CustomRole'
 
+# pylint: disable=too-many-lines
```

Comment:
nit: this is the indication that this file should be split into smaller ones.

Reply:
From my experience working with modules that did split their custom commands into multiple files, it simply added complexity.